Delicious/Extensions/UITableView+Rx.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UITableView+Rx.swift
+//  Delicious
+//
+//  Created by HoaPQ on 7/22/20.
+//  Copyright © 2020 HoaPQ. All rights reserved.
+//
+
+extension Reactive where Base: UITableView {
+    
+    func isEmpty(message: String = Constant.emptyMessage) -> Binder<Bool> {
```

Comment:
private var isEmptyBinder: Binder<(Bool, String)> {
        return Binder(self) { vc, data in
            if data.0 {
                tableView.setNoDataPlaceholder(data.1)
            } else {
                tableView.removePlaceHolder()
            }
        }
    }

chuyển qua hướng này xem dc k